### PR TITLE
feat(spans): Add schema for ingest-spans

### PR DESCRIPTION
**Pull request by phacops**

### CODE REVIEW

File: schemas/ingest-spans.v1.schema.json [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+      "type": "object",
+      "title": "span",
+      "additionalProperties": true,
+      "properties": {
+        "description": {
+          "type": "string"
+        },
+        "exclusive_time_ms": {
+          "type": "integer"
+        },
```
Comment (jjbayer) [inline schemas/ingest-spans.v1.schema.json:34]:
```suggestion
        "exclusive_time": {
          "type": "number"
        },
```

See https://github.com/getsentry/relay/blob/bd15ccff76d893117ab6a56b7e0097002733b2c2/tests/integration/test_store.py#L1242

File: schemas/ingest-spans.v1.schema.json [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "span_id": {
+          "type": "string"
+        },
+        "start_timestamp_ms": {
+          "type": "integer"
+        },
```
Comment (jjbayer) [inline schemas/ingest-spans.v1.schema.json:61]:
```suggestion
        "start_timestamp": {
          "type": "number"
        },
```

File: schemas/ingest-spans.v1.schema.json [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+        "status": {
+          "type": "string"
+        },
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "trace_id": {
```
Comment (jjbayer) [inline schemas/ingest-spans.v1.schema.json:71]:
```suggestion
        "timestamp": {
            "type": "number"
        },
        "trace_id": {
```

File: schemas/ingest-spans.v1.schema.json [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+          "type": "string"
+        }
+      },
+      "required": ["start_timestamp_ms", "trace_id"]
```
Comment (jjbayer) [inline schemas/ingest-spans.v1.schema.json:75]:
```suggestion
      "required": ["start_timestamp", "trace_id"]
```

**Review by jjbayer — CHANGES_REQUESTED** [review]

**Review by jjbayer — APPROVED** [review]

**Review by lynnagara — APPROVED** [review]

**Pull request merged**